.github/workflows/kit-bootstrap.yml

```diff
@@ -0,0 +1,152 @@
+                break;
+              else
+                echo "Control plane objects are not ready"
+              fi
+              if [[ "$time" -eq 590 ]]; then
+                echo "Timeout: Control plane objects are not ready after 10 minutes"
+                kubectl logs deploy/kit-controller -n kit
+                kubectl get pods -o wide
+                exit 1
+              fi
+              sleep 5
+            done
+            kubectl wait --for=condition=Ready pods  --all --timeout=5m
+            kubectl get secret ${GUEST_CLUSTER_NAME}-kube-admin-config -ojsonpath='{.data.config}' | base64 -d
+            cat <<-EOF | kubectl apply -f -
+              apiVersion: kit.k8s.sh/v1alpha1
+              kind: DataPlane
+              metadata:
+                name: ${GUEST_CLUSTER_NAME}-nodes
+              spec:
+                clusterName: ${GUEST_CLUSTER_NAME} # Associated Cluster name
+                nodeCount: 2
+          EOF
+            for (( time=0; time<=600; time+=5 )); do
+              dataplane_object=$(kubectl get dataplane ${GUEST_CLUSTER_NAME}-nodes  -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' 2>/dev/null)
+              if [[ "$dataplane_object" == "True" ]]; then
+                echo "Dataplane objects are ready"
+                break;
+              else
+                echo "Dataplane objects are not ready"
+              fi
+              if [[ "$time" -eq 590 ]]; then
+                echo "Timeout: Dataplane objects are not ready after 10 minutes"
+                kubectl logs deploy/kit-controller -n kit 
+                exit 1
+              fi
+              sleep 5
+            done
+            kubectl kubeconfig_guestcluster apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/release-1.10/config/master/aws-k8s-cni.yaml
+            kubectl kubeconfig_guestcluster rollout status daemonset/aws-node -n kube-system
+            for (( time=0; time<=160; time+=5 )); do
+              ready_node=$(kubectl kubeconfig_guestcluster get nodes 2>/dev/null | grep -w Ready | wc -l)
+              if [[ "$ready_node" -eq 2 ]]; then
+                echo "Dataplane nodes are ready"
+                break;
+              else
+                echo "Dataplane nodes are not ready"
+              fi
+              if [[ "$time" -eq 150 ]]; then
+                echo "Timeout: Dataplane nodes are not ready after 3 minutes"
+                exit 1
+              fi
+              sleep 5
+            done
+            kubectl kubeconfig_guestcluster create deployment nginx-deployment --image=nginx
+            kubectl kubeconfig_guestcluster rollout status deployment/nginx-deployment
+            kubectl kubeconfig_guestcluster expose deployment nginx-deployment --name=nginx-service --port=80 --target-port=80 --type=ClusterIP
+            kubectl kubeconfig_guestcluster certificate approve $(kubectl kubeconfig_guestcluster get csr | grep Pending | awk '{print $1}')
```

Comment:
Echo what I asked in https://github.com/awslabs/kubernetes-iteration-toolkit/pull/365#discussion_r1117791130
Is this doing anything useful for us?